public/client/components/System.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+import '../App.css';
+import SystemList from './SystemList';
+const axios = require('axios');
+
+const System = () => {
+  const [systemMinimum, setSystemMinimum] = useState([]);
+  const [systemRecommended, setSystemRecommended] = useState([]);
```

Comment:
Consider having some defaults here rather than just empty arrays so that you can render something if your GET request fails.

Reply:
Got it!